manifests/charts/base/templates/clusterrole.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.base.enableCRDs }}
```

Comment:
what does crd have to do with the role?

Reply:
this `enableCRDs` will mainly be used in `remote` cluster which will have no crd,  istiod and validationwebhook installed.  So think use this single flag is enough.

Reply:
But we need permission like to create `istio-ca-root-cert` in each namespace of all clusters

Reply:
we used `istio-reader` in such case, so istiod related permission is not needed.

Reply:
I still don't understand how "enableCRDs" impacts roles... aren't the two unrelated?

Reply:
see  above `this `enableCRDs` will mainly be used in remote cluster which will have no crd, istiod and validationwebhook installed. So think use this single flag is enough.  `

It means when we make `enableCRDs` as `true` , we will also need disalbe the istiod  and related..

Reply:
I removed enableCRDs flag from roles and sa as it does not hurt even on remote cluster to release concerns here. @howardjohn